shell/platform/darwin/macos/framework/Source/FlutterPlatformViews.h

```diff
@@ -0,0 +1,62 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERPLATFORMVIEWS_H_
+#define FLUTTER_FLUTTERPLATFORMVIEWS_H_
+
+#import <AppKit/AppKit.h>
+
+#import "FlutterCodecs.h"
+#import "FlutterMacros.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Wraps a `NSView` for embedding in the Flutter hierarchy
+ */
+@protocol FlutterPlatformView <NSObject>
```

Comment:
Calling an object *View when it's not an NSView is a confusing violation of convention. If it's to hold a view, call it something like ...ViewWrapper.

I'm a little confused as to what the purpose of this protocol is though. If it's only ever supposed to return one view (per the comments below), and its only method is to access the view... why not just have the view factory return a view? What problem does this intermediate class solve?

(I see that the iOS code has this too, and generally I'm in favor of matching APIs, but I'd like to understand what purpose it serves before deciding whether to mirror it.)